### ISSUE TITLE
[Feature] - Expansion to Room 3110

### ISSUE THREAD

**Issue by gaurav-karna** [pierreTklein/peer-tutoring-app#17]
Several courses like 202, 273, 206, and 250 use this room for Office Hours. I know this app is for helpdesk, but the queue mechanism would be useful in this room as well.

Perhaps this feature can advise free tutors in 3090 to help students in 3110 (like a transfer system).

**Comment by pierreTklein** [comment]
There are plans to include room / seat number for a given question, and for tutors to identify which room they are in. Is this similar to what you're referring to?

**Comment by gaurav-karna** [comment]
Yup, this would also resolve the 'expansion'. Perhaps if you want to make a unified platform instead of a whole another one for FREZCA, you can also include **building name** along with room + table/seat number. For ease of flow in the UI, maybe have the users select which session they are working with when they access the site by selecting either 'FREZCA' or 'Help-Desk' on the root page.

**Comment by gaurav-karna** [comment]
Additionally, if the table/seat number system seems too complicated (because buildings in McGill are organized differently), then you can do a ticket system ('pick-a-number' queue) that is paired with the question/topic issue as a whole. This would make things more extensible, albeit a bit less specific.

**Comment by pierreTklein** [comment]
For the extension into FRezCa / other services, I will probably follow a slack-based model, where each service is a subdomain off of the main domain.
